Pipfile

```diff
@@ -12,9 +12,14 @@ plotly = "*"
 dash_dangerously_set_inner_html = "*"
 pandas = "*"
 beaker = "*"
+scipy = "*"
 
 [dev-packages]
 flask = "*"
+black = "*"
```

Comment:
Usually I run this from a global install.  Is that wrong?  I had really high friction getting my editor to use the local `pipenv` to integrate with Black.  We can discuss if that makes sense or not.

Reply:
I'm going to remove Python Black as we have decided to install this globally as it is not really a single project's package, but should be used for code formatting across all Python applications.